web-client/src/modules/timeline/containers/TimelineViewContainer/TimelineViewContainer.tsx

```diff
@@ -195,25 +229,24 @@ const TimelineViewContainer: React.FC<TimelineViewContainerProps> = ({
     return <LoadingWrapper />;
   }
 
-  const mockRequestUser = {
-    name: 'Jon Snow',
-    rating: 4.5,
-    likes: 52,
-    distance: '5km',
-    address: '509 Gorby Lane, Jackson, FL 32065',
-    applicationPreference: 'pin',
-  };
-
   /*
     TODO: 
       Once backend changes for profile snapshot is done, instead of user={mockRequestUser},
       The Top Panel must take the user details from the request itself
   */
+  const isCav =
+    profileState.profile.applicationPreference === ApplicationPreference.cav;
+
   return (
     <>
       <TopPanel request={request} user={mockRequestUser} />;
       <TimelineList items={mockTimelineItems} currentUser={mockPin} />
```

Comment:
@btuerker why are you still mocking these? I thought you were integrating this with the timeline actions as well

Reply:
since we are a little short on time as this was to be completed today, I am taking care of this integration now.